pylint/checkers/stdlib.py

```diff
@@ -389,6 +389,22 @@ class StdlibChecker(DeprecatedMixin, BaseChecker):
             "Env manipulation functions support only string type arguments. "
             "See https://docs.python.org/3/library/os.html#os.getenv.",
         ),
+        "E1510": (
```

Comment:
I'd prefer to use a number that is not being used yet. We have `W1510` which at some point we might want to escalate to `E1510`. I believe 1519 and 1520 are free.

Reply:
Yeah, this was me misunderstanding that numbers are shared between classifiers (W, E, ..)